modules/nw-ovn-ipsec-disable.adoc

```diff
@@ -23,24 +28,34 @@ $ oc patch networks.operator.openshift.io/cluster --type=json \
   -p='[{"op":"remove", "path":"/spec/defaultNetwork/ovnKubernetesConfig/ipsecConfig"}]'
 ----
 
-. Optional: You can increase the size of your cluster MTU by `46` bytes because there is no longer any overhead from the IPsec ESP header in IP packets.
-
-. Verify that IPsec is disabled on your cluster:
+. To find the name of the OVN-Kubernetes data plane pod that exists on the `master` node in your cluster, enter the following command:
 +
 [source,terminal]
 ----
-$ oc -n openshift-ovn-kubernetes -c nbdb rsh ovnkube-master-<XXXXX> \
-  ovn-nbctl --no-leader-only get nb_global . ipsec
+$ oc get pods -n openshift-ovn-kubernetes -l=app=ovnkube-master
 ----
 +
---
-where:
+.Example output
+[source,terminal]
+----
+ovnkube-master-5xqbf                      8/8     Running   0              28m
+----
 
-`<XXXXX>`:: Specifies the random sequence of letters for a pod from the previous step.
---
+. Verify that the `master` node in your cluster has IPsec disabled by entering the following command. The command output must state `false` to indicate that the node has IPsec disabled.
 +
-.Example output
-[source,text]
+[source,terminal]
 ----
-false
-----
\ No newline at end of file
+$ oc -n openshift-ovn-kubernetes -c nbdb rsh ovnkube-master-XXXXX \// <1>
+  ovn-nbctl --no-leader-only get nb_global . ipsec 
+----
+<1> Replace `XXXXX` with the random sequence of letters for the data plane pod from the previous step.
```

Comment:
Should this use the [standard convention](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#command-syntax-for-replaceable-values) for `<user_replaceable_values>`?